boards/pba-d-01-kw2x/include/periph_conf.h

```diff
@@ -0,0 +1,494 @@
+/*
+ * Copyright (C) 2014 Freie Universität Berlin
+ * Copyright (C) 2014 PHYTEC Messtechnik GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     board_pba-d-01-kw2x
+ * @{
+ *
+ * @file
+ * @name        Peripheral MCU configuration for the phyWAVE-KW22 Board
+ *
+ * @author      Hauke Petersen <hauke.petersen@fu-berlin.de>
+ * @author      Johann Fischer <j.fischer@phytec.de>
+ * @author      Jonas Remmert <j.remmert@phytec.de>
+ */
+
+#ifndef __PERIPH_CONF_H
+#define __PERIPH_CONF_H
+
+#include "cpu-conf.h"
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+/**
+ * @name Clock system configuration
+ * @{
+ */
+#define KINETIS_CPU_USE_MCG               1
+
+#define KINETIS_MCG_USE_ERC               1
+#define KINETIS_MCG_USE_PLL               1
+#define KINETIS_MCG_DCO_RANGE             (24000000U)
+#define KINETIS_MCG_ERC_OSCILLATOR        0
+#define KINETIS_MCG_ERC_FRDIV             2
+#define KINETIS_MCG_ERC_RANGE             1
+#define KINETIS_MCG_ERC_FREQ              4000000
+#define KINETIS_MCG_PLL_PRDIV             1
+#define KINETIS_MCG_PLL_VDIV0             0
+#define KINETIS_MCG_PLL_FREQ              48000000
+
+#define CLOCK_CORECLOCK                   KINETIS_MCG_PLL_FREQ
+/** @} */
+
+
+/**
+ * @name Timer configuration
+ * @{
+ */
+#define TIMER_NUMOF                       (1U)
+#define TIMER_0_EN                        1
+#define TIMER_1_EN                        0
+#define TIMER_IRQ_PRIO                    1
+#define TIMER_DEV                         PIT
+#define TIMER_MAX_VALUE                   (0xffffffff)
+#define TIMER_CLOCK                       CLOCK_CORECLOCK
```

Comment:
Verify that this is the proper clock on the KW22. On the K60 the PIT is running on the Bus clock according to the K60 reference manual, chapter "5 Clock Distribution, 5.7 Module clocks"